steps/synopsys-dc-synthesis/configure.yml

```diff
@@ -36,6 +36,7 @@ commands:
 parameters:
   clock_period: 1.0
   design_name: undefined
+  saif_instance: undefined
```

Comment:
Can you add a comment here that says what this parameter should be?

```
parameters:
  clock_period: 1.0
  design_name: undefined
  # Path to the design instance in run.saif (e.g., tb/dut)
  saif_instance: undefined
```

I think that is what this parameter should be, right?